media/css/products/relay/components/shared.scss

```diff
@@ -0,0 +1,16 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@use '~@mozilla-protocol/core/protocol/css/components/button' as MzpButton;
```

Comment:
Do we have a convention for this yet?

Reply:
I don't think so. Maybe this can establish one?